### ISSUE TITLE
Fabric mods that were downloaded on Forge cannot receive updates

### ISSUE THREAD

**Issue by Redls07** [PrismLauncher/PrismLauncher#3104]
### Version of Prism Launcher

9.1

### Version of Qt

6.7.3

### Description of bug

If you download mods for Fabric on Forge, the launcher cannot get updates for these mods when you try to check them

### Steps to reproduce

1. Downloading Fabric mods for Forge 1.20.1 
2. Trying to check the updates of these mods
3. A warning window pops up stating that the launcher cannot find updates for these mods

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by Trial97** [comment]
Yes that is meant to be like that: you need to manually manage the mods that are not for the current version/loader

**Comment by Redls07** [comment]
Can't it be fixed?

**Comment by Redls07** [comment]
@Trial97

**Comment by Trial97** [comment]
Curseforge or Modrinth and a screenshot of that message

**Comment by Redls07** [comment]
Both
![image](https://github.com/user-attachments/assets/75cf34f9-4baf-456b-9b10-52c02b4225e4)

**Comment by Trial97** [comment]
@Redls07 can you see if you can reproduce this on the nightly build https://nightly.link/PrismLauncher/PrismLauncher/workflows/trigger_builds/develop ?

**Comment by Redls07** [comment]
Still no way to update

**Comment by Trial97** [comment]
can you share that instance or a smaller subset of it?
I tried to reproduce this but with no success.
Here is what I tried:
- create new forge 1.20.1 instance
- download a few fabric mods from the screenshot with older versions
- hit the update button
- I was prompted with the update dialog
- repeated that for both modrinth and curse

**Comment by Redls07** [comment]
https://drive.google.com/file/d/1gv6ienDaE9yq15zjn740nJicfA6kS5XC/view?usp=sharing

**Comment by Redls07** [comment]
I tried to create another instance but it was also impossible to update mods there
In the section "about Prism Launcher" it is indicated that the build date is 2024-11-03. It seems to me that the build has been around for a long time, isn't it? Maybe this is the reason?

**Comment by Trial97** [comment]
You used the previous nightly build before 9.0 was released: the fields in the meta information changed the name right before the release.
Because that build was not released a migration was not needed. You need to re-download them manually to regenerate that data.

You can follow the following tutorial:
- enable the loaders column(right click on column names then select it from that menu)
- for all the mods that the columns is empty select them and inside the "Check for Udates" menu select "Reset update metadata"
- restart prism
- Hit the check for updates (you need to do this twice once for modrinth and once for curse); you may get a retry dialog just hit no until it disappears
- then for all the mods that still have the loaders column empty please redownload them manually

**Comment by Trial97** [comment]
https://discord.com/channels/1031648380885147709/1118165588200657006/1307069628106805342
it happens on latest develop
